test/spec/ModelerSpec.js

```diff
@@ -205,14 +205,17 @@ describe('Modeler', function() {
           'already rendered {element}',
           'correcting missing bpmnElement on {plane} to {rootElement}',
           'diagram not part of bpmn:Definitions',
-          'element {element} referenced by {referenced}#{property} not yet drawn',
           'element required',
+          'element {element} referenced by {referenced}#{property} not yet drawn',
           'failed to import {element}',
+          'flow elements must be children of pools/participants',
```

Comment:
Let's ensure this is being captured when generating the translations JSON, executed via 

```
npm run collect-translations
```